plan.md

```diff
@@ -0,0 +1,71 @@
+
+# Ideas of what to include:
+
+## Filters and why:
+Main:
+- <u>**Filter by**</u> Plant id - makes it easier to look at any specific plant - say if you find a warning with a plant, it is easy to look for said plant
+- <u>**Filter by**</u> Recorded time - potentially have this as a slider - to maybe only see data from today or only from last week etc
+- <u>**Order by**</u> Last watered - not too sure about this one as it doesn't matter as some plants need to be watered more often than others
+- <u>**Filter by**</u> Plant scientific name - potentially you may only want to look at a species at a time
+
+Dropdown - <u>**Group by**</u>:
+- Botanist (name)- potentially some may be better than others so you may want to group by to see for example who is most suited to assign a plant to that comes from France
+- Origin Location - potentially to see if a warning is affecting a group of plants from the same area or for example to gather information about the ideal temperature for the plants from that area. [I feel like latitude (combined with like a longitude range) may provide more insight... but maybe not]
+- Plant scientific name - potentially if you have multiple plants of the same species, you may only care about the type so not grouping may skew the data?
+
+## Graphs
+### Live
+- Current temperature reading of every plant (I was thinking graph and expect this to be a quite up and down graph if we order by pant id, but could sort them in order of temperature so it looks better and you get a better idea of the various temperature range? <u>**Potentially ask stakeholders what they are looking to get out of this graph?**</u>)
+<br>^ I was thinking of maybe having this instead of showing current temperature, show current temperature delta (compared to its average temperature)
+- Exact same for moisture
+
+### Historical
+- for each plant, a graph for moisture, and temperature, and when watered
+<br>(potentially combine moisture and temperature on the two sides of the graph and have two lines in the graph? but may get confusing)
+
+## Layout
+Have two tabs, one for live data one for historical (live one by default/ what you see first)
+Filters on a sidebar - I would say have same exact filters for now for both
+Colours? maybe green to match plant vibes
+Above and Beyond: Incorporate pictures with the plant data
+
+### Live tab
+- Display warnings first
+- Temperature vs plant id
+- Moisture vs plant id
+
+### Historical tab
+- Just an idea idk if it's good:
+<br>Have this like a table (in terms of structure - not have an actual table - do this but putting things in containers) so maybe:
+
+(Filter sidebar over here)|Plant (scientific name)|Historical Temperature|Historical Moisture|When Watered|
+---|----|----|----|----|
+---|Heliconia schiedeana| graph | graph | graph |
+---|Cordyline fruticosa| graph | graph | graph |
+
+
+
+###### Other info
+What are warnings:
+- what is a bad soil moisture, temperature, missing water?
```

Comment:
Humidity:
even those that prefer dry conditions such as succulents, prefer the humidity to be at 40% or higher. Many thrive at levels of 60% or more.https://hydrobuilder.com/learn/low-humidity-in-plants/
**but generally 70-80% is best** https://lancaster.unl.edu/houseplants-humidity#:~:text=For%20plants%2C%20humidity%20requirements%20vary,through%20a%20process%20called%20transpiration.
Some tropical plants are used to humidity levels of up to 90% https://plnts.com/en-gb/care/doctor/temperature-and-humidity

Reply:
Temperature:
During hours of light, stick to a temperature between 25°C-30°C, depending on your lamp's light output. During dark hours you can drop this to around 20°C.
https://plagron.com/en/grow-topics/temperature#:~:text=During%20hours%20of%20light%2C%20stick,day%20as%20small%20as%20possible.

Reply:
Tropical plants might need water twice a week in the summer, 1–2 weeks in winter https://www.thesill.com/blog/drink-up
In general, houseplants' potting soil should be kept moist, but not wet. They normally need watering once or twice a week in the spring and summer, but less in the autumn and winter. https://www.gardenhealth.com/advice/houseplant-care/how-to-water-and-feed-houseplants#:~:text=In%20general%2C%20houseplants'%20potting%20soil,is%20not%20always%20the%20case.

Reply:
Soil Moisture:
The majority of flowers, trees, and shrubs require moisture levels between 21% - 40%
https://www.acurite.com/blog/soil-moisture-guide-for-plants-and-vegetables.html#:~:text=It%20is%20important%20to%20note,between%2041%25%20%2D%2080%25.